browser/resources/settings/brave_sync_page/brave_sync_subpage.ts

```diff
@@ -187,87 +201,54 @@ class SettingBraveSyncSubpage extends SettingBraveSyncSubpageBase {
           StatusAction.RETRIEVE_TRUSTED_VAULT_KEYS))
   }
 
-  /**
-  * @return {boolean}
-  * @private
-  */
-  computeSyncDisabledByAdmin_() {
-    return this.syncStatus != undefined && !!this.syncStatus.managed
+  private computeSyncDisabledByAdmin_(): boolean {
+    return this.syncStatus !== undefined && !!this.syncStatus.managed
   }
 
-  /**
-  * @return {boolean}
-  * @private
-  */
-  computeLockedSafeStorage_() {
-    return this.syncStatus != undefined && !!this.syncStatus.hasSyncWordsDecryptionError
-      && !this.syncStatus.isOsEncryptionAvailable
+  private computeLockedSafeStorage_(): boolean {
+    return this.syncStatus !== undefined &&
```

Comment:
```suggestion
    return this.syncStatus &&
```

(possibly needs a !!)

Reply:
Thanks, fixed here https://github.com/brave/brave-core/pull/27107/commits/3cb4de22c38ef674b84f1ee166deaa4e1f917c41